tools/validator-client-scripts/src/main.rs

```diff
@@ -0,0 +1,226 @@
+use crate::commands::claim_operator_reward::claim_operator_reward;
+use crate::commands::compound_delegator_reward::compound_delegator_reward;
+use crate::commands::compound_operator_reward::compound_operator_reward;
+use crate::commands::create_account::create_account;
+use crate::commands::decode_mixnode_key::decode_mixnode_key;
+use crate::commands::delegate_to_mixnode::delegate_to_mixnode;
+use crate::commands::init_contract::init;
+use crate::commands::migrate_contract::migrate;
+use crate::commands::send::send;
+use crate::commands::unbond_gateway::unbond_gateway;
+use crate::commands::unbond_mixnode::unbond_mixnode;
+use crate::commands::undelegate_from_mixnode::undelegate_from_mixnode;
+use crate::commands::update_profit_percent::update_profit_percent;
+use crate::commands::upload_contract::upload;
+use crate::commands::vesting_bond_gateway::vesting_bond_gateway;
+use crate::commands::vesting_bond_mixnode::vesting_bond_mixnode;
+use crate::commands::vesting_claim_delegator_reward::vesting_claim_delegator_reward;
+use crate::commands::vesting_claim_operator_reward::vesting_claim_operator_reward;
+use crate::commands::vesting_compound_delegator_reward::vesting_compound_delegator_reward;
+use crate::commands::vesting_compound_operator_reward::vesting_compound_operator_reward;
+use crate::commands::vesting_create_schedule::vesting_create_schedule;
+use crate::commands::vesting_delegate_to_mixnode::vesting_delegate_to_mixnode;
+use crate::commands::vesting_unbond_gateway::vesting_unbond_gateway;
+use crate::commands::vesting_unbond_mixnode::vesting_unbond_mixnode;
+use crate::commands::vesting_undelegate_from_mixnode::vesting_undelegate_from_mixnode;
+use crate::commands::vesting_update_profit_percent::vesting_update_profit_percent;
+use clap::Parser;
+use log::{error, warn};
+use network_defaults::{
+    setup_env,
+    var_names::{BECH32_PREFIX, MIX_DENOM},
+    NymNetworkDetails,
+};
+use validator_client::nymd::{self, AccountId, NymdClient, SigningNymdClient};
+
+mod commands;
+
+// we're always going to be using the signing client
+pub(crate) type Client = validator_client::nymd::NymdClient<SigningNymdClient>;
+
+#[derive(Debug, Parser)]
+pub(crate) enum Command {
+    BondGateway(commands::bond_gateway::Args),
+    BondMixnode(commands::bond_mixnode::Args),
+    ClaimOperatorReward(commands::claim_operator_reward::Args),
+    ClaimDelegatorReward(commands::claim_delegator_reward::Args),
+    CompoundOperatorReward(commands::compound_operator_reward::Args),
+    CompoundDelegatorReward(commands::compound_delegator_reward::Args),
+    Init(commands::init_contract::Args),
+    Migrate(commands::migrate_contract::Args),
+    Upload(commands::upload_contract::Args),
+    UnbondMixnode(commands::unbond_mixnode::Args),
+    UnbondGateway(commands::unbond_gateway::Args),
+    DelegateToMixnode(commands::delegate_to_mixnode::Args),
+    UnDelegateFomMixnode(commands::undelegate_from_mixnode::Args),
```

Comment:
```suggestion
    UndelegateFromMixnode(commands::undelegate_from_mixnode::Args),
```
Small nit, preventing the command from looking like `un-delegate-from-mixnode`, which looks a bit strange to me. Also fixes a typo.